rasqc/checkers/event_conditions.py

```diff
@@ -0,0 +1,455 @@
+"""Checks related to the event conditions a HEC-RAS model."""
+
+from ..base_checker import RasqcChecker
+from ..registry import register_check
+from ..rasmodel import RasModel, RasPlanHdf
+from ..result import RasqcResult, ResultStatus, RasqcResultEncoder
+
+from json import dumps
+from typing import List
+from pathlib import Path
+
+BC_PATH = "/Event Conditions/Unsteady/Boundary Conditions"
+IC_PATH = "/Event Conditions/Unsteady/Initial Conditions"
+
+
+@register_check(["ble"], dependencies=["PlanHdfExists"])
+class MeteorologyPrecip(RasqcChecker):
+    """Checker for meterology precipitation.
+
+    Reports the meterological condition precipitation applied to the
+    HEC-RAS model. Result status is 'note'.
```

Comment:
Should we prepend `RasqcChecker` classes that only note with `Note`? e.g., `NoteMeteorologyPrecip`? Just a stray thought, not a request for change.